app/models/concerns/alegre_v2.rb

```diff
@@ -336,8 +336,8 @@ def get_per_model_threshold(project_media, threshold)
       end
     end
 
-    def isolate_relevant_context(project_media, result)
-      (result["contexts"]||result["context"]).select{|x| ([x["team_id"]].flatten & [project_media.team_id].flatten).count > 0 && !x["temporary_media"]}.first
+    def isolate_relevant_context(team_id, result)
+      (result["contexts"]||result["context"]).select{|x| ([x["team_id"]].flatten & [team_id].flatten).count > 0 && !x["temporary_media"]}.first
```

Comment:
probably needs to be `[(result["contexts"]||result["context"])].flatten.select....` instead here